Pm 43 add featured pets section to home page

dotte-osu opened the pull request:
Added featured pets section (top 3 latest pets based on LastUpdated) and changed the website title.
![image](https://user-images.githubusercontent.com/71732016/155850848-8382bade-93a6-4772-b1c5-9b0d85dbf7e9.png)

dotte-osu committed:
added featured pets section

dotte-osu committed:
change site title and icon

dotte-osu committed:
changed font

dotte-osu committed:
newline

xHoudek commented on src/routes/HomePage/HomePage.js:
```diff
@@ -44,12 +62,33 @@ export default function HomePage() {
+            </div>
+          </Section>
+          <button  type='button' className='btn btn-outline-info' onClick={linkToPets}>
+            Looking for defferent match? Click here!
```
Small typo: should be "different"

dotte-osu replied:
fixed

xHoudek commented on src/routes/HomePage/HomePage.js:
```diff
@@ -44,12 +62,33 @@ export default function HomePage() {
+    if (pets) {
+      return (
+        <div className="container text-center m-3">
+          <h1 className="title text-info font-weight-bold">Meet Our New Pets!</h1>
```
Do you think this should be in all caps to match the text in the image?

dotte-osu replied:
fixed

xHoudek commented on src/routes/HomePage/HomePage.js:
```diff
@@ -44,12 +62,33 @@ export default function HomePage() {
+              <PetCard pet={pets[2]} />
+            </div>
+          </Section>
+          <button  type='button' className='btn btn-outline-info' onClick={linkToPets}>
```
I think since the purpose of this button is just to link to a different page, it should probably be `<a href='/pets' ... >` instead. You can still add the button classnames and it will look like a button. `linkToPets()` can be deleted as well

dotte-osu replied:
fixed

xHoudek commented on public/index.html:
```diff
@@ -2,14 +2,14 @@
   <head>
     <meta charset="utf-8" />
-    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <link rel="icon" href="%PUBLIC_URL%/paw.png" />
```
Thank you for adding these changes! Looks great!

xHoudek replied:
One thing I'm noticing is that it doesn't show up super well in dark mode

![image](https://user-images.githubusercontent.com/46770903/155863491-f31f56c0-21a6-4df2-9f48-8e2f0913b7da.png)

I don't think it's a big deal. Maybe we could change the color to the same one we're using as our header? Totally optional

dotte-osu replied:
thank you!

xHoudek commented on src/routes/HomePage/HomePage.js:
```diff
@@ -1,11 +1,29 @@
-import React from 'react';
-import { Section } from '../../components/Utils/Utils';
+import React, { useEffect, useState } from 'react';
+import { Section, SecondaryButton } from '../../components/Utils/Utils';
```
I think `SecondaryButton` is unused and can be removed

dotte-osu replied:
removed

xHoudek reviewed (changes requested):
Looks great! I really like the layout and everything. I just have a few minor changes and then it will be good to go

dotte-osu committed:
addressed code comment

dotte-osu commented:
Addressed code comments. Also added an icon to our "Home" title.
![image](https://user-images.githubusercontent.com/71732016/156061401-75f5c25c-ab90-43dc-a72c-86084727c942.png)

xHoudek reviewed (approved):
Newest changes look great! Loving the way the site is turning out
